src/ucp/rma/flush.c

```diff
@@ -273,8 +311,7 @@ void ucp_ep_flush_request_ff(ucp_request_t *req, ucs_status_t status)
      * this request from its pending queue
      */
     int num_comps = req->send.flush.num_lanes -
-                    ucs_popcount(req->send.flush.started_lanes)
-                    + 1;
```

Comment:
why?

Reply:
IIUC 1 was added for pending started lane, which is not counted now